econml/tests/dgp.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import numpy as np
+from econml.utilities import cross_product
+from statsmodels.tools.tools import add_constant
+
+
+class _BaseDynamicPanelDGP:
+
+    def __init__(self, n_periods, n_treatments, n_x):
+        self.n_periods = n_periods
+        self.n_treatments = n_treatments
+        self.n_x = n_x
+        return
+
+    def create_instance(self, *args, **kwargs):
+        pass
+
+    def _gen_data_with_policy(self, n_units, policy_gen, random_seed=123):
```

Comment:
Looks like maybe this and `create_instance` should be abstract?

Reply:
Fixed!

Reply:
Fixed!

Reply:
Fixed!